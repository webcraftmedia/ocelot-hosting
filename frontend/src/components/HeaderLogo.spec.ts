
import { renderSuspended } from '@nuxt/test-utils/runtime'
import Component from './HeaderLogo.vue'
import { describe, expect, it } from 'vitest'

describe('Headerlogo', () => {
  it('renders', async () => {
    const html = await (await renderSuspended(Component)).html()
    expect(html).toMatchSnapshot()
  })
})