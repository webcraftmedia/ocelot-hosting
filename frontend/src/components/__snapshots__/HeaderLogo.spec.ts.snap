// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Headerlogo > renders 1`] = `
"<div id="test-wrapper">
  <!-- eslint-disable-line @intlify/vue-i18n/no-raw-text --> Logo - ocelot + Subtitle SaaS <img src="https://flowbite.com/docs/images/logo.svg" class="h-8" alt="Ocelot Hosting Logo"><span class="self-center text-2xl font-semibold whitespace-nowrap dark:text-white">Ocelot Hosting</span>
</div>"
`;
